README.md

```diff
@@ -149,3 +150,81 @@ PostgreSQL configuration file after the benchmark completes.
 Some options can be passed on the command line and in a config file.
 Any such option passed on the command line will override the value set in the
 config file.
+
+
+# HTAP benchmark
+
+A mixed workload benchmark using a hybrid TPC-C/TPC-H schema is available in `benchmarks/htap`.
```

Comment:
iirc this is an implementation of an existing htap benchmark design. Can we reference that design here?

Reply:
It is not a straightforward implementation of another htap benchmark design. The TPC-C part is basically a port of sysbench-tpcc but with modifications required by the TPC-H part, which in turn is taken from CHbenCH. Added on top are ideas from HTAPbench.

Reply:
Your explanation would make a good comment after this line, so that one can browse through the sysbench/ChbenCH docs in order to get an idea of the benchmark.

Reply:
Yet another option: make all benchmarks use a positional argument instead of `--benchmark`.